thirdparty/faiss/faiss/invlists/OnDiskInvertedLists.cpp

```diff
@@ -13,7 +13,11 @@
 
 #include <unordered_set>
 
+#ifdef __MINGW64__
+#include "../annoy/src/mman.h"
```

Comment:
don't let faiss include annoy header file

Reply:
Currently, we need this `mman` for support mmap under windows, how about we copy it to some dedicated common folder?